src/armory/SetWagerPopup.js

```diff
@@ -58,16 +58,16 @@ class SetWagerPopup extends React.Component<Props, State> {
 
 		// Check if the wager was set. Close the popup if it is.
 		setWager(this.state.userWager, setSuccessful => {
-			if (setSuccessful) {
+			if (setSuccessful != null) {

```

Comment:
err, isn't that a boolean?

Reply:
You asked me to change it to this in another PR.

Reply:
oh, I could have been wrong lol. Is setSuccessful a boolean or an object? If it was a boolean I was definitely wrong. I thought you were doing a null check

Reply:
Like if it is suppose to be a boolean just the if statement is fine